charts/pega/README.md

```diff
@@ -867,8 +866,8 @@ Example:
 ```yaml
 # Stream (externalized Kafka service) settings.
 stream:
-  # Disabled by default, when enabled, your deployment no longer uses the "Stream" node type.
-  enabled: false
+  # from pega 8.9 Enabled by default, when disabled, your deployment no longer uses the "External kafka" configuration.
+  enabled: true
```

Comment:
I hope...upgrade testing of pega-helm is successful after this change particularly with an existing deployment with an embedded stream

Reply:
# Beginning with Pega Platform '23, enabled by default; when disabled, your deployment does not use a"Kafka stream service" configuration.
  enabled: true